server/src/server.js

```diff
@@ -291,21 +291,18 @@ app.use(function(req, res, next) {
 });
 
 app.use(function(req, res, next) {
-  const languages = accepts(req).languages() || ['en-US'];
-  l10n.init(languages).then(() => {
-    req.getText = l10n.getText;
-    req.userLocales = l10n.userLangs;
-    return l10n.getStrings().then(strings => {
-      req.messages = strings;
-      next();
-    });
-  }).catch(err => {
-    mozlog.info("l10n-error", {msg: "Error loading FTL files", description: err});
+  l10n.init().then(() => {
```

Comment:
This (`l10n.init()`) is weird in a handler, but without #2208 I guess it's the best we got.